test/integration/test_elliptic_interface_laws.py

```diff
@@ -132,21 +154,281 @@ def assign_parameters(self, gb, mortar_weight, robin_weight, rhs):
                 dir_faces = g.face_centers[0] < 1e-10
             elif g.grid_num == 2:
                 dir_faces = g.face_centers[0] > 2 - 1e-10
+            elif g.grid_num == 3:
+                dir_faces = (g.face_centers[0] < 1e-10) + (
+                    g.face_centers[0] > 2 - 1e-10
+                )
+
+            bc_val = np.zeros((g.dim, g.num_faces))
+            bc_val[0, dir_faces] = 0.1 * g.face_centers[0, dir_faces]
+            bc = pp.BoundaryConditionVectorial(g, dir_faces, "dir")
+            C = pp.FourthOrderTensor(
+                gb.dim_max(), np.ones(g.num_cells), np.ones(g.num_cells)
+            )
+            data = {
+                "bc": bc,
+                "bc_values": bc_val.ravel("F"),
+                "fourth_order_tensor": C,
+                "source": np.zeros(g.num_cells * g.dim),
+                "inverter": "python",
+            }
+            pp.initialize_data(g, d, self.kw, data)
+
+        for _, d in gb.edges():
+            mg = d["mortar_grid"]
+            MW = sps.diags(mortar_weight)
+            RW = sps.diags(robin_weight)
+            data = {
+                "mortar_weight": [MW] * mg.num_cells,
+                "robin_weight": [RW] * mg.num_cells,
+                "robin_rhs": np.tile(rhs, (mg.num_cells)),
+            }
+            pp.initialize_data(mg, d, self.kw, data)
+
+
+class TestBiotTwoGridCoupling(unittest.TestCase):
```

Comment:
While we are waiting for someone to write a user manual - haha: It seems this is the best example on how to set up the contact problem. Could you make a comment to that effect, or to a better place, somewhere in the code, maybe somewhere in biot.py, so that people like me can find it?

Reply:
Do you want it immediately? Otherwise, I think I will make a tutorial. Also the code I'm writing for the paper will probably be a good place to look when that is public

Reply:
If you make a tutorial, I will be very happy. Runscript is also very good.

Reply:
Biot with Neumann boundaries will at least become a tutorial. Contact mechanics might be a tad to complicated to fit nicely into a tutorial